assignment-5/deleteUser.js

```diff
@@ -0,0 +1,13 @@
+module.exports.deleteUser = (app, user) => {
+  app.delete('/deleteUser/:id', (req, res) => {
+    var searchID = require('./findingID');
+    var receivedID = req.params['id'];
+    var userID = searchID.findingID(user, receivedID);
```

Comment:
@JainVayam here the searchID and findID looks similar. We can change the searchID to helperMethod (because that file will contain all the helper Methods). Also, methodName should be in simple present tense. So instead of findingID, you can write as findId.